pydra/engine/tests/test_shelltask_inputspec.py

```diff
@@ -1170,6 +1172,8 @@ def test_shell_cmd_inputs_template_6():
         executable="executable", input_spec=my_input_spec, inpA="inpA", outA=False
     )
     assert shelly.cmdline == "executable inpA"
+    # checking of the command is the same
+    assert shelly.cmdline == "executable inpA"
```

Comment:
@djarecka isn't line 1176 the same as the line 1174? shouldn't we add something like `shelly.run()` between 1174 and 1176?

Reply:
actually I realized that the `run` wasn't an issue when we were checking this last week, simply repeating `cmdline` was giving a new result, it was a bug... That's why I added this to the test...